app/commands/solution/search_via_representations.rb

```diff
@@ -103,15 +98,33 @@ def search_sort
     end
   end
 
-  def to_terms(value)
-    return value.split if value.is_a?(String)
+  memoize
+  def criteria_query
+    return if criteria_query_terms.empty?
+
+    {
+      simple_query_string: {
+        query: criteria_query_terms.join(' + '),
+        fields: ["code"],
+        analyze_wildcard: true
+      }
+    }
+  end
+
+  memoize
+  def criteria_query_terms
+    criteria.filter_map do |value|
+      next if value.size < MIN_CRITERIA_LEN
 
-    [value].flatten
+      term = Infrastructure::EscapeOpensearchSimpleQueryStringTerm.(value)
+      "#{term}~#{CODE_TERM_FUZZINESS}"
```

Comment:
The fuzziness allows even smaller words to be different in one position. E.g. `class` and `closs` are considered a match. By default, opensearch will do fuzziness when it thinks it should, but it would not apply it to words as small as 10 characters. For us, I think this might be useful.